docker-database/start-db.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
+cp ../consolidated_data/submissions.csv ./DB_dockerDir/
+
+docker compose build db
+docker compose up -d
```

Comment:
```suggestion
docker compose up -d —build —force-recreate
```

should help you to avoid the tedious « kill/build/start lifecyle. You could then remove the shell scripts and only work with this command instead (that should be idempotent)

Reply:
thank you for the tip